app/src/main/java/org/mozilla/fenix/ext/BrowserState.kt

```diff
@@ -4,21 +4,36 @@
 
 package org.mozilla.fenix.ext
 
-import mozilla.components.browser.state.selector.selectedTab
+import mozilla.components.browser.state.selector.normalTabs
+import mozilla.components.browser.state.selector.selectedNormalTab
 import mozilla.components.browser.state.state.BrowserState
 import mozilla.components.browser.state.state.TabSessionState
 
 /**
- * Returns the currently selected tab if there's one as a list.
+ * Get the last opened normal tab and the last tab with in progress media, if available.
  *
- * @return A list of the currently selected tab or an empty list.
+ * @return A list of the last opened tab and the last tab with in progress media
+ * if distinct and available or an empty list.
  */
 fun BrowserState.asRecentTabs(): List<TabSessionState> {
-    val tab = selectedTab
+    // Attempt to get the selected normal tab or the last accessed normal tab
+    // if there is no selected tab or the selected tab is a private one.
+    val selectedNormalTab = selectedNormalTab
+        ?: normalTabs.maxByOrNull { it.lastAccess }
```

Comment:
nit: I mentioned this to Elise as well, I wonder if we all have different IDE settings that think the max line is shorter than what we have set on detekt. 🤔 I think this IDE plugin might help: https://github.com/detekt/detekt-intellij-plugin

This should be able to fit on one line. 

```suggestion
    val selectedNormalTab = selectedNormalTab ?: normalTabs.maxByOrNull { it.lastAccess }
```

Reply:
I intended to write this on two lines. Will make it a one liner instead.

This line will be in the getter of a new `val BrowserState.lastOpenedNormalTab` since we need to observe id/title/icon of it.